prover/src/trace/trace_lde/default/mod.rs

```diff
@@ -234,30 +230,35 @@ where
     H: ElementHasher<BaseField = E::BaseField>,
 {
     // extend the execution trace
-    #[cfg(feature = "std")]
-    let now = Instant::now();
-    let trace_polys = trace.interpolate_columns();
-    let trace_lde = RowMatrix::evaluate_polys_over::<DEFAULT_SEGMENT_WIDTH>(&trace_polys, domain);
-    #[cfg(feature = "std")]
-    debug!(
-        "Extended execution trace of {} columns from 2^{} to 2^{} steps ({}x blowup) in {} ms",
-        trace_lde.num_cols(),
-        trace_polys.num_rows().ilog2(),
-        trace_lde.num_rows().ilog2(),
-        domain.trace_to_lde_blowup(),
-        now.elapsed().as_millis()
-    );
+    let (trace_lde, trace_polys) = info_span!(
+        "Extended execution trace",
+        columns_number = field::Empty,
+        from = field::Empty,
+        to = field::Empty,
+        blowup = field::Empty
+    )
+    .in_scope(|| {
```

Comment:
Let's make the following changes here:

1. Change the name to "extend_execution_trace".
2. Use `trace.num_cols()` for `num_cols` field.
3. Use `trace.num_rows() for `from_domain_size` field.
4. Use `domain.lde_domain_size()` for `to_domain_size` field.
5. Use `domain.blowup()` for `blowup` field.